Interview/Scenes/ListContacts/ViewModels/ListContactsViewModel.swift

```diff
@@ -7,6 +7,11 @@ class ListContactsViewModel {
     
     init() { }
     
+    func getNavigationBarTitle() -> String {
+        let title = String(localizedKey: "titleText")
+        return title
+    }
```

Comment:
No Swift quando uma função com retorno e que possui apenas uma linha, não é necessário usar o return. Se quiseres simplificar essa função você pode usar da seguinte forma:
```suggestion
    func getNavigationBarTitle() -> String {
        String(localizedKey: "titleText")
    }
```